modules/virt-upgrade-pathways.adoc

```diff
@@ -13,13 +13,15 @@ You must upgrade your {product-title} to 4.7 before upgrading the minor release
 ====
 
 [id="virt-upgrade-pathways-2.4.3_{context}"]
-== Upgrading from 2.4.3 to 2.5.2
+== Upgrading from 2.4.3 to {HCOVersion}
 
-You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2.
+You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2 and up. 
 
 If your *Approval Strategy* is *Automatic*, which is the default, {VirtProductName} upgrades the z-stream automatically once you have upgraded to 2.5.0.
 
 [id="virt-upgrade-pathways-2.4.4_{context}"]
-== Upgrading from 2.4.4 to 2.5.2
+== Upgrading from 2.4.4 to {HCOVersion}
```

Comment:
Now it should be "Upgrading from 2.4.4/2.4.5 to {HCOVersion}"

Reply:
Updated here and subsequent line to "2.4.4 or 2.4.5"